### PR TITLE
Gateway comments view hooks

### PR DESCRIPTION
**Pull request by MattBudz**
### Summary

Add split button to comment feed for improved Pro parity. 

### Check List

~- [ ] Added a CHANGELOG entry~

### CODE REVIEW

File: app/assets/javascripts/tylium/behaviors.js.coffee [hunk 1]
```diff
@@ -63,7 +63,7 @@ document.addEventListener "turbolinks:load", ->
   $('.modal').on 'shown.bs.modal', ->
     $(this).find('input:text:visible:first').focus()
 
-  $('.js-try-pro').on 'click', ->
+  $('body').on 'click', '.js-try-pro', ->
```
Comment (sean-yeoh) [inline app/assets/javascripts/tylium/behaviors.js.coffee:66]: @MattBudz Can you add a comment here on why this change is needed for a smoother review process? Thanks

Comment (MattBudz) [reply app/assets/javascripts/tylium/behaviors.js.coffee:66]: We need this change in order for the upsell modal to appear. The trigger for the modal is the link in the split button, but the split button is added to the DOM via ajax. Before this change, the eventListener would bind to `.js-try-pro` on page load so it wouldn't work for the split button since the split button doesn't exist on page load.
